templates/core/core-dpl.yaml

```diff
@@ -38,6 +38,45 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+        - name: "install-certificates"
+          securityContext:
+            runAsUser: 0
+          image: {{ .Values.core.image.repository }}:{{ .Values.core.image.tag }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: ["/bin/sh"]
+          args: [
+             '-c',
+             '/usr/bin/harbor-install-cert.sh &&
+             cp -ar /var/lib/ca-certificates/* /tmp/ca-certificates' ]
+          volumeMounts:
+          - name: ca-certificates
+            mountPath: /tmp/ca-certificates
+            readOnly: false
+          - name: secret-key
+            mountPath: /etc/core/key
+            subPath: key
+          - name: token-service-private-key
+            mountPath: /etc/core/private_key.pem
+            subPath: tls.key
+          {{- if .Values.expose.tls.enabled }}
+          - name: ca-download
+            mountPath: /etc/core/ca
+          {{- end }}
+          {{- if .Values.uaaSecretName }}
+          - name: auth-ca-cert
+            mountPath: /etc/core/auth-ca/auth-ca.crt
+            subPath: auth-ca.crt
+          {{- end }}
+          {{- if .Values.internalTLS.enabled }}
+          - name: core-internal-certs
+            mountPath: /etc/harbor/ssl/core
+          {{- end }}
+          - name: psc
+            mountPath: /etc/core/token
+          {{- if .Values.caBundleSecretName }}
+{{ include "harbor.caBundleVolumeMount" . | indent 8 }}
```

Comment:
Indent value might be wrong here (I think 10 instead of 8).

Reply:
Nah, it's the same level as with normal container.

Reply:
It's not, trust me :). Just try running `helm install --dry-run --set caBundleSecretName=somecert`, and you'll get a yaml formatting failure.

Reply:
Hm, are you sure? I've changed the level of initcontainer since your first comment, so while 10 might have been true initially, I think this is correct now.